RNTester/js/PushNotificationIOSExample.js

```diff
@@ -72,11 +72,13 @@ class NotificationExample extends React.Component {
 
   _sendNotification() {
     require('RCTDeviceEventEmitter').emit('remoteNotificationReceived', {
+      remote: true,
```

Comment:
This property is added by the iOS native code for remote notification (handleRemoteNotificationReceived) and the PushNotificationIOS constructor looks for it otherwise it will not look for the aps object.